app/models/file.coffee

```diff
@@ -33,16 +33,17 @@ File = DS.Model.extend BaseModelMixin,
   isDynamicDone: DS.attr 'boolean'
   isManualDone: DS.attr 'boolean'
   isApiDone: DS.attr 'boolean'
+  isTrail: DS.attr 'boolean'
```

Comment:
Before we merge this -> we should move is trial to `user` model. I will figure this out.